ui/components/app/app-header/app-header.stories.js

```diff
@@ -5,27 +5,6 @@ export default {
   title: 'Components/App/AppHeader',
   id: __filename,
   argTypes: {
-    history: {
-      control: 'object',
-    },
-    networkDropdownOpen: {
-      control: 'boolean',
-    },
-    showNetworkDropdown: {
-      action: 'showNetworkDropdown',
-    },
-    hideNetworkDropdown: {
-      action: 'hideNetworkDropdown',
-    },
-    toggleAccountMenu: {
-      action: 'toggleAccountMenu',
-    },
-    selectedAddress: {
-      control: 'text',
-    },
-    isUnlocked: {
-      control: 'boolean',
-    },
     hideNetworkIndicator: {
```

Comment:
There doesn't appear to be any prop by this name

Reply:
Is not [here](https://github.com/MetaMask/metamask-extension/blob/a2efbb97c8014181e7de8317ad61cb5e83eff4db/ui/components/app/app-header/app-header.component.js#L1000)? Does this mean even if they are deconstructed from `this.props` it could mean that it's coming from redux?